spec/features/autocomplete_typeahead_spec.rb

```diff
@@ -47,9 +47,11 @@
         # Open the multi-image selector and choose the last one
         click_link('Change')
         find('.thumbs-list li:last-child').click
+        sleep 1
```

Comment:
Why do we need a sleep here.  Shouldn't capybara's `default_max_wait_time` be enough?